vmm_core/virt_support_x86emu/Cargo.toml

```diff
@@ -14,6 +14,7 @@ vm_topology.workspace = true
 virt.workspace = true
 x86defs.workspace = true
 x86emu.workspace = true
+iced-x86 = { workspace = true, features = ["std"] }
```

Comment:
Again I'm not sure we really want to have an iced dependency at this level. Can we remove it and use something in x86defs?

Reply:
Fixed